act-rules-format.bs

```diff
@@ -296,6 +296,19 @@ Even concepts like headings and images can be misunderstood. These terms could r
   </blockquote>
 </div>
 
+<div class=example>
+  <p>**Example:** The applicability of a rule with the page as a test target</a></p>
+  <blockquote>
+    <p>The rule applies to any page where the document element is an `html` element not embedded in another page, for example as part of an `iframe` or `object`.</p>
```

Comment:
Is it the `document element` that is not to be embedded in another page? 
That is how I read the sentence. I don't know if that is technically correct though. I hope you developer guys do know :)

Reply:
> where the document element is an `html` element not embedded in another page

this sounds a bit confusing to me: sometimes a page can be both standalone and embedded in a (possibly third-party) iframe…
Maybe we could refer to [WCAG's _Web Page_ definition](https://www.w3.org/TR/WCAG21/#dfn-web-page-s) instead?

Reply:
or alternatively we could use HTML's _browsing context_ terminology (see [_active document_](https://html.spec.whatwg.org/#active-document) and [_top-level browsing context_](https://html.spec.whatwg.org/#top-level-browsing-context)), but this may gets too technical.

Reply:
Suggestion:

> The rule applies to any page where the document element is an `html` element, and the `html` element is not embedded in another page. Examples of an `html` element embedded in another page can be as part of an `iframe` or `object` elements.